lib/Source.php

```diff
@@ -9,6 +9,20 @@
  */
 class Source extends ApiResource
 {
+    /**
+     * @return Util\Set Attributes that are nested but still updatable from
+     *    the parent class's URL (e.g. metadata).
+     */
+    public static function getNestedUpdatableAttributes()
```

Comment:
Could we add a comment somewhere in here that gives a breakdown similar to the one in this PR's description? I feel like this is a pretty difficult-to-understand complexity of the PHP library and I worry that the reasons we have `card` in here will be forgotten pretty quickly.

Reply:
👍